tests/docker/globus.debian11.Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM debian:11 
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
+
+#### install basic packages ####
+RUN apt-get update && apt-get install -y curl \
+    vim \
+    ftp \
+    telnet \
+    libcurl4-openssl-dev \
+    unzip \
+    python3 \
+    python3-distro \
+    python3-psutil \
+    python3-jsonschema \
+    python3-requests \
+    python3-pip \
+    python3-pyodbc
+
+#### Get and install iRODS repo ####
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release
+RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add -
+RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list
+RUN apt-get update
+
+#### Install icommands - used to set up, validate and tear down tests. ####
+RUN apt-get install -y irods-icommands
+
+#### Install irods-dev, cmake, clang  - used to build the connector ####
+RUN apt-get install -y irods-externals-cmake3.21.4-0
+RUN apt-get install -y irods-externals-clang*
+RUN apt-get install -y irods-dev
```

Comment:
These can probably go into 2 `RUN` commands: one to configure the iRODS packages repo, and one to install the iRODS packages. This will reduce the number of layers.

Reply:
done for all